### ISSUE TITLE
您好 能不能提供一下casia训练时的文件组织形式，感觉和另外两个不太一样

### ISSUE THREAD

**Issue by silvercherry** [VIS-VAR/LGSC-for-FAS#19]

**Comment by silvercherry** [comment]
因为看到oulu和deepfake的集有mask的操作
但是casia没有 不知道这里数据要怎么弄 
并且如果使用casia的话 train或者runner是否也需要相应修改
谢谢您

**Comment by ZGSLZL** [comment]
Hi @silvercherry , we have updated the casia.py，Casia does not require a mask
`mask = np.zeros_like(img)`

**Comment by silvercherry** [comment]
> Hi @silvercherry , we have updated the casia.py，Casia does not require a mask
> `mask = np.zeros_like(img)`

yes i have seen that,but i want to konw how to load the train_list and val_list
